tools/hive/hadoop-hive/scripts/hive-bootstrap.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -x
+#save all env vars .bashrc for ssh sessions
+printenv | cat >> /root/.bashrc
+
+# hadoop bootstrap
+/etc/hadoop-bootstrap.sh -d
```

Comment:
There is no `-d` flag in the `hadoop-bootstrap.sh` script above. Is this intentional?

Reply:
removed `-d`